lib/objectid.js

```diff
@@ -0,0 +1,12 @@
+var ObjectId;
+
+try {
+  ObjectId = require('mongodb').ObjectId;
+} catch(e) {
+  if (e.code === 'MODULE_NOT_FOUND')
+    throw e;
```

Comment:
@superkhau AFAICT, this silently swallows any `mongodb` parse errors, which does not look like a good thing to me. Could you please explain what is the rationale for this, why we need to give `require('mongodb').ObjectId` errors a special treatment?